10/10/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Xml;
+
+namespace Mass
+{
+    class Program
+    {
+
+        static void Main()
+        {
+            Console.WriteLine("Введите целочисленные первый член последовательности и ее разность");
+            int firstNum = int.Parse(Console.ReadLine());
+            int step = int.Parse(Console.ReadLine());
+            int[] nums = new int[10];
+            nums[0] = firstNum;
+            for (int i = 1; i < 10; i++)
+                nums[i] = nums[i - 1] + step;
+            var pr = new Program();
+            pr.OutputMass(nums);
+            Console.WriteLine("Введите целое число, на которое увеличится каждый элемент массива");
+            int plus = int.Parse(Console.ReadLine());
+            pr.Summ(nums, plus);
+            pr.OddNums(nums);
+            Console.WriteLine("Введите степень, на которую увеличить каждый элемент массива");
+            int degreeOfMass = int.Parse(Console.ReadLine());
+            pr.Degree(nums, degreeOfMass);
+        }
+        public void OutputMass(int[] nums)
+        {
+            Console.WriteLine("Ваша последовательность:");
+            for (int i = 0; i < 9; i++)
+                Console.Write(nums[i] + ", ");
+            Console.WriteLine(nums[9]);
+        }
+        public void Summ(int[] nums, int plus)
+        {
+            Console.WriteLine($"Ваша последовательность увеличенная на {plus}:");
+            for (int i = 0; i < 9; i++)
+                Console.Write((nums[i] + plus) + ", ");
+            Console.WriteLine(nums[9] + plus);
+        }
+        public void OddNums(int[] nums)
```

Comment:
Отделяйте методы в классе пустой строкой. Так легче читать код.
```suggestion
        }
        
        public void OddNums(int[] nums)
```